migrations/webid.js

```diff
@@ -0,0 +1,55 @@
+import fs from 'node:fs'
+import { fileURLToPath } from 'node:url';
+import { dirname, join } from 'node:path';
+import { rdfParser } from 'rdf-parse'
+import { arrayifyStream } from 'arrayify-stream'
+import { createVocabulary } from 'rdf-vocabulary'
+import { write } from '@jeswr/pretty-turtle'
+import { Store, DataFactory } from 'n3'
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+const filePath = join(__dirname, '../catalog-data.ttl');
+const ex = createVocabulary('http://example.org#', 'webid')
+
+function changeSubject(quad, subject) {
+  return DataFactory.quad(subject, quad.predicate, quad.object)
+}
+
+function changeObject(quad, object) {
+  return DataFactory.quad(quad.subject, quad.predicate, object)
+}
+
+const textStream = fs.createReadStream(filePath, { encoding: 'utf8' })
+const fromStream = rdfParser.parse(textStream, {
+  contentType: 'text/turtle',
+});
+const dataset = new Store(await arrayifyStream(fromStream))
+
+const quads = await arrayifyStream(dataset.match(null, ex.terms.webid, null))
+console.log('records with webid ', quads.length)
+
+for (const quad of quads) {
+  const tmpId = quad.subject
+  const webid = quad.object
+
+  if (tmpId.equals(webid)) continue
+
+  dataset.delete(quad)
+  dataset.add(changeSubject(quad, webid))
+
```

Comment:
```suggestion
```

This could be removed since it will match on subject anyways and be taken of there.
At the same time it is one-time migration script which has no tests so maybe better not to touch it for such a minor tiding up.